packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -465,6 +470,12 @@ export function createCacheSnapshotKey(odspResolvedUrl: IOdspResolvedUrl): ICach
 	return cacheEntry;
 }
 
+export function snapshotWithLoadingGroupIdSupported(
+	config: IConfigProvider,
+): boolean | undefined {
+	return config.getBoolean("Fluid.Container.UseLoadingGroupIdForSnapshotFetch2");
```

Comment:
Now that I think about it, this is not important for this PR, but I wonder if we should have used `getString` instead of `getBoolean` in case we come up with a new "snapshot" fetch type

Reply:
Sorry didn't get it. We are just reading the value of feature gate here. Snapshot type is not important here.?